subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -2027,7 +2027,7 @@ static int cmd_wifi_ap_enable(const struct shell *sh, size_t argc,
 	    cnx_params.security == WIFI_SECURITY_TYPE_EAP_PEAP_GTC ||
```

Comment:
Could you fix the commit message, the latter wrong API name should be `wifi_set_enterprise_creds` (now they are the same).